WBooks/AppDelegate.swift

```diff
@@ -32,7 +32,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         NetworkActivityLogger.shared.startLogging()
         
         window = UIWindow(frame: UIScreen.main.bounds)
-        window?.rootViewController = WBooksNavigationController(rootViewController: LoginViewController())
+        window?.rootViewController = LoginViewController()
         window?.makeKeyAndVisible()
         
         // MARK: NavigationBar global styles config
```

Comment:
I think the lines below (the ones that start with `UINavigationBar.appearance()`) here are not necessary anymore, since the status bar appearance is defined by each controller now.